lib/reverse_words.rb

```diff
@@ -1,6 +1,50 @@
 # A method to reverse each word in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2) where n is the length of my_words?
```

Comment:
Since the `while` loops are nested, yes this is O(n<sup>2</sup>). You are traversing each word and then each character within the word.